dashboard/app/models/pd/facilitator_program_registration.rb

```diff
@@ -0,0 +1,286 @@
+# == Schema Information
+#
+# Table name: pd_facilitator_program_registrations
+#
+#  id         :integer          not null, primary key
+#  user_id    :integer          not null
+#  form_data  :text(65535)
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#  teachercon :integer
+#
+# Indexes
+#
+#  index_pd_fac_prog_reg_on_user_id_and_teachercon        (user_id,teachercon) UNIQUE
+#  index_pd_facilitator_program_registrations_on_user_id  (user_id)
+#
+
+require 'state_abbr'
+
+class Pd::FacilitatorProgramRegistration < ActiveRecord::Base
+  LOCATIONS = [
```

Comment:
If it's all the same, include the state names in these. There's a decent sized Philadelphia, MS